actions/platform-test/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "platform-test",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
```

Comment:
we have out own license https://github.com/sberdevices/plasma/blob/master/LICENSE.txt